docs/user_guide/ttl.md

```diff
@@ -14,10 +14,34 @@ store-version, after the push finishes, but before swapping the read traffic ove
 replay phase where the last N seconds of buffered real-time data is replayed. The replayed data gets overlaid on top of
 the data from the Full Push.
 
-This can be leveraged in order to achieve a TTL behavior. By scheduling periodic empty pushes and configuring how far 
-back to replay the real-time data (N), one can control the TTL parameters. The time to live is defined by N, which acts 
-as a "minimum TTL", while the "maximum TTL" is N + delay between each empty push. For example, if you schedule a daily 
-empty push, and N = 6 days, then the oldest data in your store will be at least 6 days old, and at most 7 days old.
+This can be leveraged in order to achieve a TTL behavior. Currently, there are two options that leverages this mechanism. 
+
+### Empty push
```

Comment:
Hmm... I think this Empty Push section and the TTL Repush section should go to the end, probably under Usage. It should start with use cases, then the usage.